File: group 19/day 22/homework/homework5.py
#1)Calculate the sum of digits of a number entered by the user.
#გამოთვალეთ მომხმარებლის მიერ შეყვანილი რიცხვის რიცხვების ჯამი


numbers = input("შემოიტანეთ რიცხვები: ")
total = 0
for num in numbers:
    total += int(num)

File: group 25/level 084/homework/task 5/homework5.js
function calculateSum(n) {
  let sum = 0;
  switch (true) {
    case n >= 1:
      for (let i = 1; i <= n; i++) {
        sum += i;
      }
      break;
    default:
      sum = 0;
  }
  return sum;
}